site/content/how-to/monitoring/dashboard.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "NGINX Plus Dashboard"
+description: "Learn how to view the NGINX Plus dashboard to see real-time metrics."
+weight: 200
+toc: true
+docs: "DOCS-000"
+---
+
+{{<custom-styles>}}
+
+## Overview
+
+The NGINX Plus dashboard offers a real-time live activity monitoring interface that shows key load and performance metrics of your server infrastructure. The dashboard is enabled by default for NGINX Gateway Fabric deployments that use NGINX Plus as the data plane. The dashboard is available on port 8765.
+
+To access the dashboard:
+
+1. Use port-forwarding to forward connections to port 8080 on your local machine to port 8765 on the NGINX Gateway Fabric pod (replace `<nginx-gateway-fabric-pod>` with the actual name of the pod).
+
+    {{< note >}}8080 is just an example local port. You can specify any unused port that you want.{{< /note >}}
```

Comment:
nit:
if you use port 8765  instead of 8080 here, would still need this note?
why use different ports?

Reply:
Couple reasons:

1. To show that you don't have to use the same port (some newbies may assume they have to be the same)
2. Pick an easy port for localhost that is probably not taken.

Though I'm not opposed to keeping them the same, if preferred.

Reply:
> Though I'm not opposed to keeping them the same, if preferred.

8080 is a special port, so I'd recommend choosing a different one